controllers/goharbor/harborcluster/cache/redis.go

```diff
@@ -1,13 +1,25 @@
 package cache
 
 import (
-	goharborv1alpha2 "github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha2"
+	"github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha2"
 	"github.com/goharbor/harbor-operator/pkg/lcm"
 )
 
-type RedisReconciler struct {
+type RedisController struct {
 }
 
-func (r RedisReconciler) Reconcile(harborCluster *goharborv1alpha2.HarborCluster) (*lcm.CRStatus, error) {
+func (r *RedisController) Apply(harborcluster *v1alpha2.HarborCluster) (*lcm.CRStatus, error) {
```

Comment:
how about the params `harborcluster` moved to struct `MinIOController`? and set it by `NewCacheController`.